discovery_child_development/pipeline/openalex_metaflow.py

```diff
@@ -0,0 +1,168 @@
+"""
+works pipeline
+--------------
+
+A pipeline that takes a list of concept IDs and years, and outputs OpenAlex API results.
+
+The thought behind this is to break the results into manageable yearly chunks. For a given year
+and high level concept, the output works may be well over 2GB in size when saved to json.
+"""
```

Comment:
Please add instructions for running the flow here (ie, how to run it from terminal), for example something like this:

```
Usage:

python discovery_child_development/pipeline/openalex_metaflow.py --production
```